Corona-Warn-App/src/main/java/de/rki/coronawarnapp/qrcode/ui/QrCodeScannerFragment.kt

```diff
@@ -239,38 +241,13 @@ class QrCodeScannerFragment : Fragment(R.layout.fragment_qrcode_scanner), AutoIn
                 QrCodeScannerFragmentDirections.actionUniversalScannerToSubmissionDeletionWarningFragment(
                     scannerResult.restoreRecycledTestRequest
                 )
-            is CoronaTestResult.TestPending ->
```

Comment:
We need to keep this here!
The idea is that `CoronaTestQRCodeHandler` should still provide all possible cases ,but the consumer of these results such as `Recycle-Bin` , `MainActivity`(Deep-link) or `Scanner` they decide what to do with these results

Reply:
Did you comment on the wrong class?
But I can add the restored test to the Event 'RestoredTest', so the consumer can use this to decide what to do next. I guess this would be a little bit better than creating an event for every test result...